compose/backend.yml

```diff
@@ -224,23 +225,23 @@ services:
     networks:
       - ccd-network
 
-# Uncomment this section to enable ccd-shared-database with Postgres version 11.
-#  ccd-shared-database-v11:
-#    build: ../database-v11
-#    healthcheck:
-#      interval: 10s
-#      timeout: 10s
-#      retries: 10
-#    environment:
-#      DB_USERNAME:
-#      DB_PASSWORD:
-#      POSTGRES_HOST_AUTH_METHOD: trust
-#    ports:
-#      - 5055:5432
-#    volumes:
-#      - ccd-docker-ccd-shared-database-data-v11:/var/lib/postgresql/data
-#    networks:
-#      - ccd-network
+  #This section should be commented when the pr is merged in master.
```

Comment:
This will be commented on the final merge version

Reply:
this has been updated 👍